### DISCUSSION TITLE
Hyperion LED strips not showing up

### DISCUSSION THREAD

**Discussion by CAuguston** [milo526/homebridge-tuya-web#313] [Q&A]
In the Smart Life app the LED strip shows up and works. However the light never gets imported via the Tuya-Web plugin. Is there support for the LED light strips?

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.

**Reply by CAuguston** [reply]
Got credentials
{   'access_token': 'Removed',
    'expires_in': 864000,
    'refresh_token': 'Removed',
    'token_type': 'bearer'}
Getting devices
Got devices
{   'header': {'code': 'SUCCESS', 'payloadVersion': 1},
    'payload': {   'devices': [   {   'data': {   'brightness': '300',
                                                  'color_mode': 'colour',
                                                  'color_temp': 18647,
                                                  'online': True,
                                                  'state': 'false'},
                                      'dev_type': 'light',
                                      'ha_type': 'light',
                                      'icon': 'https://images.tuyaus.com/smart/icon/ay1530625888062OnLeq/15759822201fe70f38bb2.png',
                                      'id': '00202114500291d74c99',
                                      'name': 'Living room lamp'},
                                  {   'data': {   'brightness': '894',
                                                  'color_mode': 'white',
                                                  'color_temp': 18647,
                                                  'online': True,
                                                  'state': 'true'},
                                      'dev_type': 'light',
                                      'ha_type': 'light',
                                      'icon': 'https://images.tuyaus.com/smart/icon/ay1530625888062OnLeq/15759822201fe70f38bb2.png',
                                      'id': '00202114f4cfa256cb24',
                                      'name': 'Bedroom Lamp'},
                                  {   'data': {},
                                      'dev_type': 'scene',
                                      'ha_type': 'scene',
                                      'id': 'OAzyHqpqG0IW16hk',
                                      'name': 'Bedroom lamp white'},
                                  {   'data': {},
                                      'dev_type': 'scene',
                                      'ha_type': 'scene',
                                      'id': 'HBOjIHsYvtpeMogV',
                                      'name': 'Bedroom Lamp Purple'}],
                   'scenes': []}}
![IMG_5498](https://user-images.githubusercontent.com/45345498/129081193-a98f3f26-efa7-4c75-b56c-814119da54e3.jpg)
![thumbnail_IMG_5496](https://user-images.githubusercontent.com/45345498/129081195-9571d24a-7f0b-43b8-966c-140330f16599.png)
